<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterContractDialog</class>
 <widget class="QDialog" name="RegisterContractDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RegisterContractDialog{border:1px solid rgb(68,217,199);background-color:white;}
QLineEdit{border:1px solid rgb(214,214,214);border-radius:4px;padding: 0px 8px 0px 8px;font: 12px &quot;微软雅黑&quot;;}
QLabel{font: 12px &quot;微软雅黑&quot;;}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>合约文件：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>最多init费用：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>基本费用：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QToolButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>460</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="initCostLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="baseCostLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ALP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>预计init费用：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="expectedCostLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>注册合约</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>12</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{background-image:url(:pic2/close5.png);background-repeat: repeat-xy;background-position: center;background-attachment: fixed;background-clip: padding;border-style: flat;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
